api.go

```diff
@@ -50,16 +50,18 @@ type StatusResponse struct {
 
 func writeSuccess(w http.ResponseWriter) {
 	resp := &StatusResponse{Ok: true}
-	out, _ := json.Marshal(resp)
+	out, _ := json.MarshalIndent(resp, "", "")
 	w.WriteHeader(http.StatusOK)
 	_, _ = w.Write(out)
+	_, _ = w.Write([]byte("\n"))
 }
 
 func writeError(w http.ResponseWriter, status int, err error) {
 	resp := &StatusResponse{Ok: false, Message: err.Error()}
-	out, _ := json.Marshal(resp)
+	out, _ := json.MarshalIndent(resp, "", "")
```

Comment:
Any particular reason why the code uses `MarshalIndent` now, but doesn't set indentation?

Reply:
This makes the JSON a bit more pretty-printed, which makes it easier to read when there's big error messages.

Reply:
We could set the indent but idk, doesn't really matter